postprocessing/janus-pp-rec.1

```diff
@@ -24,6 +24,9 @@ Only print the supported target file extensions per codec  (default=off)
 .BR \-j ", " \-\-json
 Only print JSON header  (default=off)
 .TP
+.BR \-e ", " \-\-ext
+Only print extensions  (default=off)
```

Comment:
As discussed in the comments, I't call this something like "extended report", rather than "extensions". In the future we may not only have the extension in there.